bitrise.yml

```diff
@@ -154,6 +163,10 @@ workflows:
         title: Get project version
         inputs:
           - content: bitrise/scripts/get_project_version.sh --allow-snapshots
+    - script@1.1.5:
```

Comment:
why are we installing the client but not doing anything?

Reply:
subsequent steps in the pipeline require the client to be installed